### ISSUE TITLE
Unhandled JS Exception: Cannot read property 'systemShouldAdvance' of undefined

### ISSUE THREAD

**Issue by barnesheaton** [nHiRanZ/react-native-kenburns-view#10]
### Running into this error

![Error](https://cdn.pbrd.co/images/HOyAEUA.png)

It seems to be occurring when the parent component is still mounted but the KenBurnsImage component isn't currently being displayed. This is all within a `<Modal>`

# Code

```
<Modal
  animationType={'slide'}
  transparent={true}
  visible={this.props.workoutStore.starting && !this.props.workoutStore.remote}
  onRequestClose={this.props.workoutStore.cancel}
  onDismiss={this.props.workoutStore.ready}
>
  <View style={{flex: 1}}>
  ...
    <KenBurnsImage
      tension={4}
      friction={25}
      imageWidth={Dimensions.get('window').width}
      imageHeight={Dimensions.get('window').height}
      placeholderSource={require('../../../img/landing.png')}
      sourceUri={require('../../../img/landing.png')}
    />
  ...
  </View>
</Modal>

```

### Environment

| software         | version
| ---------------- | -------
| react-navigation | 2.17.0
| react-native     | 0.57
| node             | 9.11.2
| yarn      | 1.7.0

**Comment by jgutierro** [comment]
same here!

I noticed that is package is using this robound dependence:

`"rebound": "^0.0.15"`

Last version of rebound is 0.1.0

Maybe this error is fix in 0.1.0 version...

**Comment by jgutierro** [comment]
@barnesheaton  have you found a solution or workaround?
